frontend/src/utils/glueSerializer.js

```diff
@@ -1,6 +1,89 @@
 export const glueSerializer = tables => {
   const result = { databases: [] };
 
+  const ARRAYSTRUCTPREFIX = "array<struct<";
+  const STRUCTPREFIX = "struct<";
+  const SCHEMA_INVALID = "Column schema is not valid";
+  const ALLOWEDTYPES = [
+    "bigint",
+    "char",
+    "double",
+    "float",
+    "int",
+    "smallint",
+    "string",
+    "tinyint",
+    "varchar"
+  ];
+
+  const columnMapper = c => {
```

Comment:
I think we could probably do with a test for this? also it's quite hard to follow what's happening and some tests would help with that

Reply:
can we break this up a bit to make it more readable?

Reply:
done

Reply:
done